src/python-common/ceph/tests/test_service_spec.py

```diff
@@ -62,6 +62,7 @@ def test_parse_host_placement_specs(test_input, expected, require_network):
         ('3 data[1-3]', "PlacementSpec(count=3, host_pattern='data[1-3]')"),
         ('3 data?', "PlacementSpec(count=3, host_pattern='data?')"),
         ('3 data*', "PlacementSpec(count=3, host_pattern='data*')"),
+        ("count:2 max-per-host:4", "PlacementSpec(count=2, max_per_host=4)"),
```

Comment:
Would be great to have a special case for "just allow co-located daemons. I don't care about details" maybe `PlacementSpec(max_per_host=-1)` or so as `max_per_host` is a bit special for RGW.